spec/routing/hourly_rates_routing_spec.rb

```diff
@@ -19,30 +19,40 @@
 
 require 'spec_helper'
 
-describe HourlyRatesController, :type => :routing do
-  describe "routing" do
-    it { expect(get('/projects/blubs/hourly_rates/5')).to route_to(:controller => 'hourly_rates',
-                                                               :action => 'show',
-                                                               :project_id => 'blubs',
-                                                               :id => '5') }
+describe HourlyRatesController, type: :routing do
+  describe 'routing' do
+    it {
+      expect(get('/projects/blubs/hourly_rates/5')).to route_to(controller: 'hourly_rates',
+                                                                action: 'show',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.